xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -951,7 +944,7 @@ struct UserProfileDetailModel: ModelProtocol {
     ) -> Update<Self> {
         update(
             state: state,
-            action: .refresh(forceSync: true),
+            action: .refresh(forceSync: false),
```

Comment:
We shouldn't need to sync after resolution, because resolution comes as result of a sync.